.github/workflows/e2e.yml

```diff
@@ -115,3 +82,16 @@ jobs:
       - name: Run tests
         run: |
           make test-e2e
+
+      - name: Export kind logs
+        if: always()
+        run: |
+          mkdir -p /tmp/kind/logs
+          kind export logs --loglevel=debug /tmp/kind/logs
+
+      - name: Upload kind logs
+        if: always()
+        uses: actions/upload-artifact@v3
+        with:
+          name: kind-logs-${{ env.JOB_NAME }}-${{ github.run_id }}
+          path: /tmp/kind/logs
```

Comment:
Path is the source directory. Copied from the [doc](https://github.com/actions/upload-artifact#inputs):
```
    # A file, directory or wildcard pattern that describes what to upload
    # Required.
    path:
```

Reply:
Logs are kept in github for a period of time, and according to the doc, by default the time is set in the repository settings.
```
    # Duration after which artifact will expire in days. 0 means using default retention.
    # Minimum 1 day.
    # Maximum 90 days unless changed from the repository settings page.
    # Optional. Defaults to repository settings.
```

My intent is to download the logs within the next 2 hours of finishing the test. But maybe it would be interesting to extend the duration to 12 hours so that we can collect the logs from a nightly run. WDYT?

Reply:
I misread! Ok, the upload-artifact just does that. Great!

Reply:
> In this path, GH automatically archives the logs?

👀  https://github.com/actions/upload-artifact#where-does-the-upload-go